src/main/java/com/denizenscript/depenizen/bukkit/events/mythicmobs/MythicMobsSpawnEvent.java

```diff
@@ -0,0 +1,106 @@
+package com.denizenscript.depenizen.bukkit.events.mythicmobs;
+
+import com.denizenscript.denizencore.objects.*;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.depenizen.bukkit.objects.mythicmobs.MythicMobsMobTag;
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import io.lumine.xikage.mythicmobs.adapters.AbstractLocation;
+import io.lumine.xikage.mythicmobs.api.bukkit.events.MythicMobSpawnEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+
+public class MythicMobsSpawnEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // mythicmob mob spawns
+    // mythicmob <mob> spawns
+    //
+    // @Regex ^on mythicmob [^\s]+ spawns$
+    //
+    // @Switch in:<area> to only process the event if it occurred within a specified area.
+    //
+    // @Triggers when a MythicMob spawns.
+    //
+    // @Context
+    // <context.mob> Returns the MythicMobTag that is spawning.
+    // <context.entity> Returns the EntityTag for the MythicMob.
+    // <context.location> Returns a LocationTag of where the MythicMob will spawn.
+    // <context.from_spawner> Returns true if the mob was from a spawner.
+    // <context.spawner_location> Returns a LocationTag of the spawner that spawned the mob.
+    //
+    // @Plugin Depenizen, MythicMobs
+    //
+    // @Group Depenizen
+    //
+    // -->
+
+    public MythicMobsSpawnEvent() {
+        instance = this;
+    }
+
+    public static MythicMobsSpawnEvent instance;
+    public MythicMobSpawnEvent event;
+    public MythicMobsMobTag mythicmob;
+    public LocationTag location;
+
+    @Override
+    public boolean couldMatch(ScriptPath path) {
+        return (path.eventLower.startsWith("mythicmob")
+                && (path.eventArgLowerAt(2).equals("spawns")));
```

Comment:
Again, this is misformatted - check the formatting of any `couldMatch` method in Denizen.